github/src/main/java/com/novoda/github/reports/service/network/GithubApiService.java

```diff
@@ -71,6 +72,7 @@
             @Query("per_page") int perPageCount
     );
 
+    @Deprecated
```

Comment:
Why was this deprecated? 😕 
Is there any alternative for someone that still wants to get this functionality?